plugins/synaptics-prometheus/synaptics-prometheus.quirk

```diff
@@ -38,3 +38,14 @@ InstallDuration = 2
 Plugin = synaptics_prometheus
 InstallDuration = 2
 
+[USB\VID_06CB&PID_0123]
```

Comment:
0123 sure looks suspicious...

Reply:
I thought the same thing, but I reached out to Vincent at Synaptics and then noticed the exact same ID in fprint
https://gitlab.freedesktop.org/libfprint/libfprint/-/commit/128d809227bbef1d3bd8e7c7e1f9b68a335db252